cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Store.umc.tsx

```diff
@@ -5,24 +5,26 @@ import React, { createContext, ReactNode, useReducer } from 'react';
 import { STORE_SECTION_FORM } from '../reducers/Form.reducer';
 import { StoreDispatch } from '../../shared/types/types';
 import umcReducer from '../reducers/Wizard.reducer';
+import { UNMANAGED_CLUSTER_FIELDS } from '../../views/unmanaged-cluster/unmanaged-cluster-wizard-page/UnmanagedCluster.constants';
 
 const initialState = {
     [STORE_SECTION_FORM]: {
         // Cluster settings basic
-        CLUSTER_NAME: '',
+        [UNMANAGED_CLUSTER_FIELDS.CLUSTER_NAME]: 'unmanged-cluster',
 
         // Cluster Settings advanced
-        CONTROL_PLANE_NODES_COUNT: 1,
-        WORKER_NODES_COUNT: 0,
-        CLUSTER_PROVIDER: 'calico',
+        [UNMANAGED_CLUSTER_FIELDS.CONTROL_PLANE_NODE_COUNT]: 1,
+        [UNMANAGED_CLUSTER_FIELDS.WORKER_NODE_COUNT]: 0,
+        [UNMANAGED_CLUSTER_FIELDS.CLUSTER_PROVIDER]: 'kind',
 
         // Cluster Network Settings
-        IP_ADDRESS: '127.0.0.1',
-        HOST_PORT_MAPPING: '80',
-        NODE_PORT_MAPPING: '80',
-        CLUSTER_SERVICE_CIDR: '100.64.0.0/13',
-        CLUSTER_POD_CIDR: '100.96.0.0/11',
-        CLUSTER_PROTOCOL: 'tcp',
+        [UNMANAGED_CLUSTER_FIELDS.CNI_PROVIDER]: 'calico',
```

Comment:
I would suggest we do something similar to what I did with the K8sProviders.constants.tsx file and create a set of constants for the CNI providers. THe file could be named CniProviders.constants.tsx